test/test.css

```diff
@@ -5393,14 +5393,67 @@
 }
 #sandbox .popover.upcycle-hover_tooltip {
   background-color: #FFF;
-  border-radius: 5px;
-  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.25);
+  border-radius: 3px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.3);
+  padding: 8px 16px;
+  max-width: 248px;
+  border: none;
 }
+#sandbox .popover.upcycle-hover_tooltip .popover-title,
 #sandbox .popover.upcycle-hover_tooltip .popover-content {
-  font-family: Arial, Helvetica, sans-serif;
-  font-size: 12px;
+  font-family: inherit;
   color: #666;
-  padding: 10px;
+  padding: 0;
+  font-size: 11px;
+  line-height: 14px;
+  background-color: #FFF;
```

Comment:
@cmcilhenny the `background-color` here is the style that is "sharpening" some of the border corners in our module. Can we remove this and opt for the using the `background-color` on the `.popover.upcycle-hover_tooltip`? There should not be a discernible difference since the colors are the same